modules/getting-started/p_configure-appserver.adoc

```diff
@@ -0,0 +1,54 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to configure my JBoss EAP instance
+
+[id="configure-appserver"]
+= Configuring the {appserver_name} instance
+
+The {appserver_name} servlet application requires a final procedure before it is secured by {project_name}. Use the following procedure to configure the {appserver_name} instance.
```

Comment:
Maybe change to
"The {appserver_name} servlet application requires additional configuration before it is secured by {project_name}. Use the following procedure to configure the {appserver_name} instance."